anycloud/node/src/index.ts

```diff
@@ -1,6 +1,6 @@
+export { datastore, ds } from './datastore';
 export const PORT = 8088; // http://img2.wikia.nocookie.net/__cb20080604102014/starwars/images/3/3b/8t88.png
 
 export default {
-  PORT,
+  PORT
```

Comment:
nit: any reason to get rid of the trailing comma? Also, why do we `export const` the port and provide it in `default`? Doesn't that do the same thing?

Reply:
So this is one piece that I was a little confused about but didn't want to make a fuss for now.

But what this `export default { ... }` does is create an object that is what's actually imported if you just `import anycloud from 'anycloud';`

Ideally, this would also have `datastore` and `ds` properties in it so then you can do `anycloud.ds['foo'] = 'bar';` and that would also work.

The idea behind this is so you don't have to write `import * as anycloud from 'anycloud';`

Maybe we should just delete this because it doesn't seem to bring much value.

Reply:
Yes, I have leave only the named exports